package debt_simpl;

message TestReq {
  optional string msg = 1;
}

message TestRes {
  optional string msg = 1;
}

service DebtSimplifier {
  rpc Test(TestReq) returns (TestRes) {}
}
